components/ui/GoogleAdsense/index.tsx

```
@@ -0,0 +1,103 @@
+  responsive,
+  slot,
+  setDisplayAd,
+  adClient,
+}: GoogleAdsenseContainerProps): JSX.Element => {
+  const adRef = useRef<HTMLModElement>(null);
+  const observer = useRef<MutationObserver | null>(null);
+
+  useEffect(() => {
+    if (
+      adRef.current &&
+      adRef.current.children &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)